### ISSUE TITLE
Add credits command

### ISSUE THREAD

**Issue by louisdh** [louisdh/terminal#91]
A `credits` command would print info about the app (e.g. version number), including the names of all the contributors. The output would be ASCII art, ideally generated with a script (since the version number and contributors change quite often).

Some inspiration (created at http://patorjk.com/software/taag/#p=display&f=Big&t=OpenTerm):
```
   ____                _______                  
  / __ \              |__   __|                 
 | |  | |_ __   ___ _ __ | | ___ _ __ _ __ ___  
 | |  | | '_ \ / _ \ '_ \| |/ _ \ '__| '_ ` _ \ 
 | |__| | |_) |  __/ | | | |  __/ |  | | | | | |
  \____/| .__/ \___|_| |_|_|\___|_|  |_| |_| |_|
        | |                                     
        |_|                                     
```

**Comment by IMcD23** [comment]
I like this idea. Version number can be retrieved from `Info.plist`. Should we start storing the contributors in another plist that's bundled in the app?

**Comment by louisdh** [comment]
Implemented in 988c085acdd339ba5a645ca8f46267a9c428112d.
Contributors are stored in `authors.json` (I might rename that to `contributors.json`).
The `credits` command also displays the version and build number.
